src/server/pfs/server/api_server.go

```diff
@@ -832,7 +832,7 @@ func (a *apiServer) RunLoadTest(ctx context.Context, req *pfs.RunLoadTestRequest
 
 func (a *apiServer) runLoadTest(pachClient *client.APIClient, branch *pfs.Branch, specStr string, seed int64) error {
 	spec := &pfsload.CommitsSpec{}
-	if err := yaml.UnmarshalStrict([]byte(specStr), spec); err != nil {
+	if err := serde.DecodeYAML([]byte(specStr), spec); err != nil {
```

Comment:
Our serde library uses `gopkg.in/yaml.v3` under the hood, so this change gets all of our yaml parsing using the same library.